### DISCUSSION TITLE
Raspap connected without internet

### DISCUSSION THREAD

**Discussion by JOMOGO** [RaspAP/raspap-webgui#1172] [Q&A]
After a clean install of raspbian lite and following this https://raspap.com/#quick I restarted the pi and connected with the ap but got the message "Connected without internet"

I could not find a solution online does somebody have any Idea why it does not work?

What I use:
Raspberry pi 3B
Raspbian lite bullseye also tried the gui version
tested ethernet cable straight from the router tried with wifi dongle and with both at the same time

**Comment by billz** [comment]
This usually indicates a 'self-assigned' IP address, ie. your device is not receiving an IP from the AP. This can happen for several reasons, discussed in the FAQ with troubleshooting steps:

https://docs.raspap.com/faq/#noip
